js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-features-section.js

```diff
@@ -103,32 +94,34 @@ export default function PaidAdsFeaturesSection( {
 		>
 			<Section.Card>
 				<Section.Card.Body>
-					<Flex
-						className="gla-paid-ads-features-section__content"
-						align="center"
-						gap={ 9 }
-					>
-						<FlexBlock>
-							<Section.Card.Title>
-								{ __(
-									'Drive more sales with Performance Max',
-									'google-listings-and-ads'
-								) }
-							</Section.Card.Title>
-							<div className="gla-paid-ads-features-section__subtitle">
-								{ __(
-									'Reach more customers by advertising your products across Google Ads channels like Search, YouTube and Discover. Set up your campaign now so your products are included as soon as they’re approved.',
-									'google-listings-and-ads'
-								) }
-							</div>
-							<FeatureList
-								hideBudgetContent={ hideBudgetContent }
-							/>
-						</FlexBlock>
-						<FlexItem>
-							<CampaignPreview />
-						</FlexItem>
-					</Flex>
+					<VerticalGapLayout size="medium">
+						<Flex
+							className="gla-paid-ads-features-section__content"
+							align="center"
+							gap={ 9 }
+						>
+							<FlexBlock>
+								<Section.Card.Title>
+									{ __(
+										'Drive more sales with Performance Max',
+										'google-listings-and-ads'
+									) }
+								</Section.Card.Title>
+								<div className="gla-paid-ads-features-section__subtitle">
+									{ __(
+										'Reach more customers by advertising your products across Google Ads channels like Search, YouTube and Discover. Set up your campaign now so your products are included as soon as they’re approved.',
+										'google-listings-and-ads'
+									) }
+								</div>
+								<FeatureList />
+							</FlexBlock>
+							<FlexItem>
+								<CampaignPreview />
+							</FlexItem>
+						</Flex>
+
+						{ ! hideBudgetContent && <FreeAdCredit /> }
```

Comment:
This correctly retains the current behavior for the $500 credit, which meets the expectations for this PR. However, the current behavior as implemented actually doesn't actually make sense, because the `hideBudgetContent` prop is only `true` when this component is rendered by the `SetupPaidAds` component when there is no Google Ads connection ([ref](https://github.com/woocommerce/google-listings-and-ads/blob/db9d822f614f8860110d07d8d14c8014bd829e1a/js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js#L173)).

Now that Google Ads has moved to the first step of the onboarding flow, this will always be true. I'll create a new ticket to address this logic and we can move this forward in the meantime.

Reply:
👍